action/protocol/execution/evm/evmstatedbadapter.go

```diff
@@ -530,6 +544,8 @@ func (stateDB *StateDBAdapter) Prepare(rules params.Rules, sender, coinbase comm
 	if !rules.IsBerlin {
 		return
 	}
+	// Clear out any leftover from previous executions
+	stateDB.accessList = newAccessList()
```

Comment:
this is not necessary? we are creating a new `StateDBAdpator` for each tx

Reply:
for safety, it's best to add

Reply:
What is the relationship with eip-1153?

Reply:
> for safety, it's best to add

for what kind of safety? We need to be clear what we want to achieve.

Reply:
the code has updated in
https://github.com/ethereum/go-ethereum/pull/26003/files#diff-c3757dc9e9d868f63bc84a0cc67159c1d5c22cc5d8c9468757098f0492e0658cR1073-R1074

i think we missed it